### PR TITLE
feat: Add a window/dialog after successful transaction actions

### PR DESCRIPTION
**Pull request by devchenyan**
issue: https://github.com/Magickbase/neuron-public-issues/issues/362

https://github.com/nervosnetwork/neuron/assets/11756869/94c9ae5e-cd6e-4abd-96bd-85a704b84359

### CODE REVIEW

**Comment by Danie0918** [comment]
@homura @yanguoyu Please have a review.

File: packages/neuron-ui/src/locales/en.json [hunk 1]
```diff
@@ -771,6 +775,16 @@
       "cancel": "Cancel",
       "next": "Next"
     },
+    "deposit-rules": {
+      "minimum-deposit": "Minimum Deposit",
+      "single-compensation-cycle": "Single Compensation Cycle",
+      "single-compensation-cycle-description": "180 Epoch (About 30 days)",
```
Comment (Keith-CY) [inline packages/neuron-ui/src/locales/en.json:781]:
```suggestion
      "single-compensation-cycle-description": "180 Epochs (About 30 days)",
```

File: packages/neuron-ui/src/locales/en.json [hunk 2]
```diff
@@ -771,6 +775,16 @@
...
+      "withdraw": "Withdraw",
+      "withdraw-description": "Any Time",
```
Comment (Keith-CY) [inline packages/neuron-ui/src/locales/en.json:783]:
```suggestion
      "withdraw-description": "Anytime",
```

File: packages/neuron-ui/src/locales/zh-tw.json [hunk 3]
```diff
@@ -737,17 +739,50 @@
...
+      "address": "地址",
+      "amount": "數量",
+      "cell-from-cellbase": "來自 Cellbase",
+      "cancel": "取消",
+      "next": "下一步"
+    },
+    "deposit-rules": {
+      "minimum-deposit": "最低存入額",
+      "single-compensation-cycle": "單一補償週期",
+      "single-compensation-cycle-description": "180 個Epochs（約 30 天）",
```
Comment (Keith-CY) [inline packages/neuron-ui/src/locales/zh-tw.json:775]:
```suggestion
      "single-compensation-cycle-description": "180 個 Epochs（約 30 天）",
```

File: packages/neuron-ui/src/locales/zh.json [hunk 4]
```diff
@@ -707,6 +707,8 @@
       "estimated-apc": "预计年化锁定补贴率",
       "estimated-apc-tooltip": "预计年化锁定补贴率",
       "deposit-amount": "存入金额 (CKB)",
+      "attention": "请注意： 一旦存入 Nervos DAO，您的存款将被锁定至少 180 个epochs（约 30 天）。",
```
Comment (Keith-CY) [inline packages/neuron-ui/src/locales/zh.json:710]:
```suggestion
      "attention": "请注意： 一旦存入 Nervos DAO，您的存款将被锁定至少 180 个 Epochs（约 30 天）。",
```

File: packages/neuron-ui/src/locales/zh-tw.json [hunk 5]
```diff
@@ -708,6 +708,8 @@
       "estimated-apc": "預計年化鎖定補貼率",
       "estimated-apc-tooltip": "預計年化鎖定補貼率",
       "deposit-amount": "存入金額 (CKB)",
+      "attention": "請注意： 一旦存入 Nervos DAO，您的存款將被鎖定至少 180 個epoch（約 30 天）。",
```
Comment (Keith-CY) [inline packages/neuron-ui/src/locales/zh-tw.json:711]:
```suggestion
      "attention": "請注意： 一旦存入 Nervos DAO，您的存款將被鎖定至少 180 個 Epochs（約 30 天）。",
```

File: packages/neuron-ui/src/locales/zh.json [hunk 6]
```diff
@@ -764,6 +768,16 @@
       "cancel": "取消",
       "next": "下一步"
     },
+    "deposit-rules": {
+      "minimum-deposit": "最低存入额",
+      "single-compensation-cycle": "单补偿周期",
+      "single-compensation-cycle-description": "180 个Epochs（约 30 天）",
```
Comment (Keith-CY) [inline packages/neuron-ui/src/locales/zh.json:774]:
```suggestion
      "single-compensation-cycle-description": "180 个 Epochs（约 30 天）",
```

File: packages/neuron-ui/src/widgets/Icons/SortV2.svg
Comment (Keith-CY) [inline packages/neuron-ui/src/widgets/Icons/SortV2.svg:1]: It's better to name the icon according to its usage.

**Review by Keith-CY — APPROVED** [review]

**Review by homura — APPROVED** [review]

**Comment by silySuper** [comment]
1.when less than 1hour, here shows 0 hours.
截屏2024-04-19 15 45 07

截屏2024-04-19 15 20 39
截屏2024-04-19 15 20 44


2.UI is different from design
actual:
截屏2024-04-19 15 23 31

截屏2024-04-19 15 26 00

design:
截屏2024-04-19 15 48 23

**Comment by silySuper** [comment]
@devchenyan please check this~

**Comment by silySuper** [comment]
In No.2 tips is lost.

**Comment by silySuper** [comment]
截屏2024-04-30 14 24 48
this does not fix @devchenyan

**Review by silySuper — APPROVED** [review]

**Pull request merged**